plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -351,14 +354,15 @@ private class DokkaDescriptorVisitor(
         )
     }
 
-    fun CallableMemberDescriptor.createDRI(wasOverridenBy: DRI? = null): Pair<DRI, DRI?> =
-        if (kind == CallableMemberDescriptor.Kind.DECLARATION || overriddenDescriptors.isEmpty())
-            Pair(DRI.from(this), wasOverridenBy)
-        else
-            overriddenDescriptors.first().createDRI(DRI.from(this))
```

Comment:
This was wrong as it was passing only neighbouring DRI to the next level of recursion. We want the most upper and lower DRIs. Moreover it was wrongly interpreted then as an `inheritedFrom` insted of `overridenBy` in 361 line

Reply:
Anyway, I think it is nice to use this function for `visitPropertyDescriptor` as well. I would also change the extra `InheritedFunction` to `InheritedCallable` and then change presentation of property/function row to something like this

![obraz](https://user-images.githubusercontent.com/32793002/92941533-ca7d6d00-f450-11ea-82ca-4968734ef7a0.png)

@Kordyjan @sellmair @kamildoleglo what do you think on that?